docker-compose.yml

```diff
@@ -239,8 +244,10 @@ services:
       - _APP_REDIS_PORT
       - _APP_REDIS_USER
       - _APP_REDIS_PASS
```

Comment:
Can we deprecate these variables now in favour of the new ones ?

Reply:
Not yet. We should continue supporting them for community edition where you might want a single server. I also need to add back the database ones. We can rethink later how to deal with all of them.

Reply:
okay 👍